package.json

```diff
@@ -41,28 +41,26 @@
   },
   "homepage": "https://github.com/Skyscanner/backpack-android#readme",
   "devDependencies": {
-    "bpk-svgs": "^5.19.0",
-    "bpk-tokens": "^27.0.12",
+    "bpk-svgs": "^5.19.1",
+    "bpk-tokens": "^27.1.0",
```

Comment:
can we also bump dependency for "replace-in-file" from https://github.com/Skyscanner/backpack-android/pull/68/